platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -148,8 +149,10 @@ define(
 
                 // Enforce minimum/maximum positions
                 function enforceExtrema() {
+                    // Check for user preference on splitPane width
+                    var userWidthPreference = window.localStorage[$attrs.alias];
```

Comment:
Can you prefix the localStorage key, e.g. `var preferenceKey = "mctSplitPane-" + $attrs.alias`?  That should prevent any possible future confusion or conflicts with other uses of localStorage.